new/LIT7021BaryaAganentGudale.xml

```diff
@@ -0,0 +1,59 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7021BaryaAganentGudale" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Protective prayer against bāryā, demons, desk, gudāle and budā</title>
```

Comment:
Dasak? I wonder, if bāryā and dasak are better to be seen as demonic beings, as well. I think, they should be given in capitals in the title? Cf. LIT6856PPHemamaBarya (Prayer against Bāryā and Legewon). E.g. "Bāryā, demons, Dasak (or: Dask), gudāle (or: gudāle) and budā".

Reply:
OK